File: level20/homework/task6.py
#6) მომხმარებელმა უნდა შეიყვანოს რიცხვები, სანამ -1-ს არ შეიყვანს. საბოლოოდ გამოიანგარიშოს შეყვანილი რიცხვების საშუალო
number=int(input("enter a number"))
count=0
sum=0
while number !=-1:
    count +=1
    sum +=number
    number=int(input("enter a number"))
print(sum / number)

File: level 065/homework/task6.js
//  ფუნქცია კუბს გამოთვლის და გამოიტანს
function cubeNumber() {
    const input = document.getElementById("numInput");
    const output = document.getElementById("numOut");

    const num = Number(input.value);

    if (isNaN(num)) {
        output.textContent = "Please enter a valid number.";
        return;
    }

    const cube = Math.pow(num, 3); // ან: num * num * num

    output.textContent = `Cube: ${cube}`;
}

//  ღილაკზე დაჭერისას გამოვიძახოთ cubeNumber()
document.getElementById("cubeBtn").addEventListener("click", cubeNumber);

/*

- input.value იღებს შეყვანილ რიცხვს.
- Number() გარდაქმნის სტრიქონს რიცხვად.
- Math.pow(num, 3) ან num ** 3 → ითვლის კუბს.
- შედეგი გამოდის <p> ელემენტში.
*/
